### ISSUE TITLE
Unable to run `srcd sql` on home directory

### ISSUE THREAD

**Issue by gomesfernanda** [src-d/engine#206]
Whenever trying to run `srcd sql` on my home directory, I get the following error:

```bash
rpc error: code = Unknown desc = SQL query failed: dial tcp 172.18.0.4:3306: connect: connection refused
```

**srcd version:**

* srcd cli version: v0.8.0
* docker version: 1.39
* srcd daemon version: v0.8.0

**Mac version:** 10.14.2

**Comment by se7entyse7en** [comment]
@smacker are you referring to this [here](https://github.com/src-d/engine/pull/195#discussion_r254648083)?

**Comment by smacker** [comment]
yes

**Comment by dpordomingo** [comment]
Yes. it will be also needed some work to provide a better output, but with your PR the UX is enhanced :D

**Comment by se7entyse7en** [comment]
@gomesfernanda this [issue](https://github.com/src-d/engine/pull/195) has been closed and should also address this one. If everything works as expected the only thing to be handled better is the log message as described in [this](https://github.com/src-d/engine/issues/211) other issue.

Could you try the newly merged code? Or do you need a new release first?

**Comment by carlosms** [comment]
The issue has been reproduced and tested by us, I think there is no need for @gomesfernanda to configure the building environment. We will include the fix in the next release, so I'll close this one now.

Fixed by #195.

**Comment by dpordomingo** [comment]
@gomesfernanda already tried the merged code running engine from sources; and it worked ;)
